tools/ldb_test.py

```diff
@@ -340,6 +340,19 @@ def testDumpLoad(self):
         self.assertFalse(self.dumpDb(
             "--db=%s --create_if_missing" % origDbPath, dumpFilePath))
 
+        # Dump and load with BlobDB enabled
+        blobParams = " ".join(["--enable_blob_files", "--min_blob_size=1024",
```

Comment:
Since `min_blob_size` is 1024 and the values are really small (`y1`, `y2`, etc.) I don't think this test case will actually create any blob files.

We would also want to add coverage for `put` and `batchput` with blob files enabled.

Reply:
It would be nice to add checks that make sure blob files are actually generated by the appropriate test cases.

Reply:
Please don't forget to add some tests for `put` and `batchput`.